src/providers/response-types/kaltura-flavor-asset-list-response.ts

```diff
@@ -0,0 +1,38 @@
+import {KalturaFlavorAsset} from './kaltura-flavor-asset';
+const {BaseServiceResult} = KalturaPlayer.providers.ResponseTypes;
+
+export class KalturaFlavorAssetListResponse extends BaseServiceResult {
+  totalCount?: number;
+  data: Array<KalturaFlavorAsset> = [];
+
+  constructor(responseObj: any) {
+    super(responseObj);
+    if (!this.hasError) {
+      this.totalCount = responseObj.totalCount;
+      if (this.totalCount! > 0) {
+        this.data = this.setDataWithoutDuplicates(responseObj.objects);
+        //responseObj.objects.map((flavorAsset: KalturaFlavorAssetArgs) => this.data.push(new KalturaFlavorAsset(flavorAsset)));
+      }
+    }
+  }
+
+  private setDataWithoutDuplicates(flavors: Array<any>): Array<KalturaFlavorAsset> {
+    const data = [];
+    // sort flavors by height
+    const sortedFlavors = flavors.sort((a: {height: number}, b: {height: number}) => {
+      return a.height < b.height ? 1 : -1;
```

Comment:
you can also use 
return (b.height - a.height)

Reply:
yep, i guess that's another way to do it.. i took it from qualitySettings comp in ui. i thought it would be good to be aligned.